src/Raven.Studio/wwwroot/Content/css/styles-common.less

```diff
@@ -2058,6 +2058,16 @@ ul.properties {
     border-color: @brand-warning !important;
 }
 
+.passive-node-info {
+    width: 100%;
+    padding: @gutter;
+    margin-bottom: @gutter !important;
+    background-image: url(../img/pages/about/passive-state-bg.svg);
```

Comment:
are you sure this is correct url? 
You moved that from `css/pages/about.less` to `css/styles-common.less` so I think we need another `../` in url, right?

Reply:
As discussed now with @kwiato 
the path is not relative to the LESS file location,
it is relative to the CSS file generated from the LESS
(and it is working)

Reply:
When doing review I don't checkout your or anyone else code and sometimes I do blind guesses. If you know something is working fine - just say so and we move forward. :) No need for proves, etc. :)